PoSSapi/src/Application/Reservations/Commands/CreateReservationCommand.cs

```diff
@@ -0,0 +1,44 @@
+using AutoMapper;
+using MediatR;
+using PoSSapi.Application.Common.Interfaces;
+using PoSSapi.Domain.Entities;
+using PoSSapi.Domain.Enums;
+
+namespace PoSSapi.Application.Reservations.Commands;
+
+public class CreateReservationCommand : IRequest<Guid>
+{
+    public Guid Id { get; init; }
+    public string Name { get; init; }
+    public DateTime Time { get; init; }
+    public int NumOfPeople { get; init; }
```

Comment:
You're not gonna like it, but Time should be validated that it's after now and NumOfPeople that it's >0 ;D